flutter_common/lib/data/generation_helpers/write_json_example.main.dart

```diff
@@ -29,8 +29,8 @@ Future<void> main() async {
   final runResult = BenchmarkRunResult(
     throughput: 123.45,
     accuracy: Accuracy(
-      value: 123.45,
-      string: '123.45%',
+      normalized: 0.123,
+      formatted: '123.45%',
```

Comment:
`0.123` would be `12.3%`.

Reply:
Done.

I don't see the point, though. These values never-ever go our of this file. The string may as well be empty and all numeric values may as well be zero or anything else. This is only for automatic json schema generation, it was never meant to be read by anyone.